src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceBase.cs

```diff
@@ -77,9 +77,6 @@ public async Task<T> ExecuteScalarQueryAsync<T>(string query, CancellationToken
         /// <inheritdoc/>
         public abstract IEnumerable<DataSourceObjectMetadata> GetChildObjects(DataSourceObjectMetadata parentMetadata, bool includeSizeDetails = false);
 
-        /// <inheritdoc/>
-        public abstract void Refresh();
```

Comment:
Removed function from interface and abstract since it's not called outside KustoDataSource

Reply:
We may need it here because its a method any other data store we later add will need to implement as well. Something to double check.

Reply:
Agree. That looks like an override for any other DataSource we implement.